tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,402 @@
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__CE_Event_Cost_Currency_Symbol
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+ * Extension main class, class begins loading on init() function.
+ */
+class Tribe__Extension__CE_Event_Cost_Currency_Symbol extends Tribe__Extension {
+
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 */
+	public function construct() {
+		// Only works with/after version 4.5 as previous versions included the field
+		$this->add_required_plugin( 'Tribe__Events__Community__Main', '4.5' );
+
+		$this->set_url( 'https://theeventscalendar.com/extensions/community-events-cost-currency-symbol/' );
+	}
+
+	/**
+	 * Extension initialization and hooks.
+	 */
+	public function init() {
+		add_action( 'admin_init', array( $this, 'add_settings' ) );
+		add_action( 'tribe_events_community_section_before_cost', array( $this, 'display' ) );
+	}
+
+	/**
+	 * Add options to Tribe settings page.
+	 *
+	 * @see Tribe__Extension__Settings_Helper
+	 */
+	public function add_settings() {
+		require_once dirname( __FILE__ ) . '/src/Tribe/Settings_Helper.php';
+
+		// Setup fields on the settings page
+		$setting_helper = new Tribe__Extension__Settings_Helper();
+
+		// List of allowed symbols
+		$fields['tribe_ext_ce_cost_currency_symbols'] = array(
+			'type'            => 'text',
+			'label'           => esc_html__( 'Event Cost: Allowed Currency Symbols', 'tribe-extension' ),
+			'tooltip'         => esc_html__( 'Pipe-separated list of allowed currency symbols. (Once this option is set and saved, you will be able to pick the Default Currency Symbol for new events.) The drop-down will display these in the order you enter them. Example 1: $|€|£|Fr|¥|CNY|C$|AU$. Example 2: USD|EUR|GBP|JPY|CNY|CAD|AUD', 'tribe-extension' ),
+			'validation_type' => 'textarea',
+		);
+
+		// only allow modifying after the List exists (to be able to pick from)
+		if ( tribe_get_option( 'tribe_ext_ce_cost_currency_symbols' ) ) {
+			$fields['tribe_ext_ce_cost_currency_symbol_default'] = array(
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'Event Cost: Default Currency Symbol', 'tribe-extension' ),
+				'tooltip'         => esc_html__( 'The default currency symbol for new events (does not apply when editing existing events). Leave blank to not set a default.', 'tribe-extension' ),
```

Comment:
Oh I am sorry, all of my comments assumed this was adding settings to the Default Content tab. That said, all of our tabs have a pretty unified design language. So modeling your tooltips and labels after those on the default Content page might make sense.

One thing none of our labels ever do is use colons. So you would never see "Event Cost: Currency Symbol". And also we usually keep our tooltips as short as possible, and suggest a default format rather than an example. So reviewing the other settings areas and modeling this one after them could be awesome. But, if you feel that's not worth the effort no worries.